catalog/operator-tests.yaml

```
@@ -21,6 +21,30 @@
   weekly_test:
     platform: replicated-azure
     test_script_params: --parallel 1
+- id: commons-operator
+  display_name: Commons Operator
+  git_repo: commons-operator
+  spec:
+    node-count: 3
```

I think a single node should be sufficient
```suggestion
    node-count: 1
```